### ISSUE TITLE
Basic Med Settings allow Person to Die after Revive timer has ran out, Even if Fully Bandaged to Stop Bleeding

### ISSUE THREAD

**Issue by armyinf0703** [acemod/ACE3#1531]
As title States, we healed a few people tonight, but was waiting on an medic with EPI as everyone else had ran out, We fully stopped all bleeding, and just need morphine/ EPI to wake up, But they died at the 210 seconds mark (Set in our module). If they are not bleeding anymore, shouldn't they be allowed to go past that 210 mark?

ACE Medical > Medical Settings
Medical Level: Basic
Medics setting: Disable medics
Enable Litter: Yes
Life time of litter: 180
Enable Screams: No
Player Damage: 1.5
AI Damage: 1
AI Unconsciousness: Disabled
Prevent instant death: Yes
Bleeding coefficient: 0.5
Pain coefficient: 1
Sync Status: Yes

ACE Medical > Set Medic Class
List: N/A
Is Medic: None

ACE Medical > Revive Settings
Enable Revive: Player Only
Max Revive time: 210
Max Revive lives: -1

**Comment by gienkov** [comment]
No, you have 210 seconds to *wake them up*, otherwise they die.

**Comment by armyinf0703** [comment]
That's what I am saying, something like an CPR button to extend  that time be built in like it was in arma2 ace? If they are fully bandaged, you should be able to extend the time,

**Comment by SonixApache** [comment]
The group I play with uses the revive system in combination with the advanced medical system to prevent people from immediately dying if shot while down. However, even if they are fully bandaged, and not in pain, if they don't wake up they die, which is a damn shame. 

Would it be possible to only apply this timer when the patient's heart is stopped, or the blood level is critically low?

**Comment by J-Ziegler** [comment]
A stable patient shouldn't die. Still bleeding? Sure. Heavy blood loss? Maybe. Pain? Debatable. However, someone who only needs epinephrine dying is just annoying.

Since there's no way to actually tell how badly off the patient is, it's hard to prioritize casualties. It also leads to gaming the system by giving a patient epi to thwart the timer, which is silly even if it isn't resolved by fixing this issue.

**Comment by TheConen** [comment]
Suggestion: Add an option to the basic revive module like "Reset/Extend counter on:" with options like "Never", "Successful CPR", "Successful CPR + no bleeding" etc.. This way people can choose which style they want to play - hardcore-like with people dying if MedEvac is not in time or not so hardcore-like with people having also a chance if the MedEvac needs a few minutes longer.

In out community we currently use CSE with a 30min timer so there is enough time to get the wounded to a safe LZ, let the MedEvac fly in, load them in, fly back to base, heal them and fly them back. During that time, they just lie around because noone can do anything for them and they can only be healed at base. With an option to reset or extend the counter, it would be possible to reduce the countdown to e.g. 5min while still having enough time for the whole MedEvac thing - and instead of just lying around in the corner, people have to take care of the wounded, give them constantly CPR etc.. Furthermore, people who got left behind don't need to wait that long before they die.

**Comment by AtlantisThief** [comment]
This caused also some confusion on our community. We expected that, as soon as someone is not bleeding, his blood pressure and pulse are normal/ok, we should be able to keep him like that for an extensive time. Even giving morphine against the pain and epinephrin to make them wake up did not help for patients, that has once reached a "critical" state (very low blood pressure and/or pulse). After that timer then passed, they died. 
I expected that once someone is not in a critical state, the timer should simply stop and that guy can be revived at any point, unless his values drop into that critical state again.

**Comment by GloryAndPain** [comment]
I'm in the same boat.  Would like to set a damage level/bleed level that equates to death, but would like to have that death timer only apply if the patient is still bleeding or heart has stopped.  
Or have death tied to blood loss specifically, and threshold for passing out from blood loss able to be set.

I would like to be able to force more small arms-engagements to unconscious injuries that need prompt attention without having to resort to using the disable-death unconscious timer system which makes getting hit by a tank silly.  The current settings as I understand them either mean it's too much chance of death, and a lot of trial and error to get any level of consistent unconscious states, or using the unconscious state timer but being able to medical people who should rightly be dead.

**Comment by Glowbal** [comment]
As Gienkov said:
> No, you have 210 seconds to wake them up, otherwise they die.

Closing this. If the question is not answered, please create a new ticket (or ask on the Bohemia forums).

I am looking into CPR to extend the revive timer while a unit is in revive state.
